chips/sam4l/src/pm.rs

```diff
@@ -487,22 +500,69 @@ macro_rules! mask_clock {
         let val = (*PM).$field.get() | ($mask);
         (*PM).$field.set(val);
     });
+
+    ($module:ident: $field:ident & $mask:expr) => ({
```

Comment:
This looks identical to the block above except that it's conditioned on `&` instead of `|` -- when would the conjunction be true when the disjunction wasn't already true?